src/gui/LayoutMapper.cpp

```diff
@@ -19,52 +12,44 @@ LayoutMapper::~LayoutMapper()
 }
 
 
-void LayoutMapper::configureFromSettings(int numPages, Settings* settings)
+void LayoutMapper::configureFromSettings(size_t numPages, Settings* settings)
 {
 	XOJ_CHECK_TYPE(LayoutMapper);
 
+	auto isPairedPages = settings->isShowPairedPages();
```

Comment:
The variables here have different types, and the function names are not clear enough to accurately guess the types. The use of `auto` here is hurting readability; I would keep some of the explicit type annotations here.

Reply:
The variables have the same type, because I changed the type of the corresponding member variable to bool, since it was only used as bool and it is bad practice to check if an int is unequal to 0 just by wrighting the variable into the if statement. And a variable starting with or containing a verb is normally a bool. I changed the variable name to "showPairPages". This should be more readable.

Reply:
Ok i misunderstood you, i changed it now, is this better?
I still think it's not important for readability, whether a number is an int, unsigned, long or size_t.
Most of the time, it does not even matter if it's a float or a double. That what matters is the purpose, and for what the variable is used. For this pourpose an "auto const(&)" is fully sufficient. It also has the big advantage, that you can exchange the type without having troubles to change the type everywhere it is used or copied.